careers/ladder.md

```diff
@@ -28,73 +28,183 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
                   M1  > M2  > M3  > M4 > E1 > E2
```

Comment:
not sure if it's intended or not, but looks like the whitespace surrounding the `>`s differ between `M`s and `E`s